webapp/pom.xml

```diff
@@ -103,29 +103,29 @@
             </configuration>
          </plugin>
          <plugin>
-            <groupId>org.mortbay.jetty</groupId>
-            <artifactId>maven-jetty-plugin</artifactId>
-            <version>6.1.10</version>
-            <configuration>
-               <webXml>${project.build.directory}/jetty/WEB-INF/web.xml</webXml>
-               <webAppConfig>
-                  <contextPath>/${webapp.name}</contextPath>
-                  <!--enable reloading static resources -->
-                  <overrideDescriptor>src/test/resources/override-web.xml</overrideDescriptor>
-                  <extraClasspath>target/classes;../api/target/classes;../web/target/classes</extraClasspath>
-               </webAppConfig>
-               <scanIntervalSeconds>10</scanIntervalSeconds>
-               <scanTargets>
-                  <scanTarget>../api/target/classes</scanTarget>
-                  <scanTarget>../web/target/classes</scanTarget>
-               </scanTargets>
-               <systemProperties>
-                   <systemProperty>
-                       <name>OPENMRS_INSTALLATION_SCRIPT</name>
-                       <value>${OPENMRS_INSTALLATION_SCRIPT}</value>
-                   </systemProperty>
-               </systemProperties>
-            </configuration>
+             <groupId>org.mortbay.jetty</groupId>
+             <artifactId>jetty-maven-plugin</artifactId>
+             <version>8.0.0.M0</version>
```

Comment:
http://mvnrepository.com/artifact/org.eclipse.jetty/jetty-maven-plugin says the latest version is 9.2.10.v20150310 (note the groupId has changed as well) Could you please try it instead?